### PR TITLE
Feat/search post

### PR DESCRIPTION
**Pull request by Haroenv**
This PR is made so that comments can be made 👍

### CODE REVIEW

File: _posts/2017-05-09-searching-for-packages.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+
+This is where [Algolia](https://www.algolia.com/), the company I work for, comes into play. As a developer-focused search engine API, we were already fan of Yarn and wanted a cool hack for our end of the year (2016) gift.
+
+On the 22nd December of 2016, Algolia engineers via [yarnpkg/website#322](https://github.com/yarnpkg/website/pull/322) then ten days later it got merged and instant-search for JavaScript packages was available on our website. At first there was no package details pages.
+
+Early 2017, the core team of Yarn and Algolia team met for a one day brainstorming in London. The goal was to think about the package details page and evolutions of the search interface. Algolia proposed [invision](https://www.invisionapp.com/) design views of what search could be and from that we drafted a [master plan](https://gist.github.com/vvo/c801fb8b653eda9fb17de60987476b5e)!
+ 
+ ## What makes it different
```
Comment (Haroenv) [inline _posts/2017-05-09-searching-for-packages.md]: remove leading space

File: _posts/2017-05-09-searching-for-packages.md [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+
+The stargazers and commit activity are fetched with the GitHub API without a specific logged in user; we do this because it proved to have high enough limits for most usages. I only hit into situations where we reached the API limit when needing to reload a lot when developing on the detail page. If this proves to happen more often, we could add a 
+
+## What it means for the community
+
+What we currently made for the Yarn website is a proof of concept. We believe in the power of community and allow anyone to use the search index made by us to be used for logical or off-the-charts 🆒 ideas.
+
+Right now we don't have an automated system for giving out API keys yet, but you can request one by [filling out the form](//todo: the form). You will then get an Algolia API key for the search index with `search` and `browse` as capabilities. You can read more about how to use this on the [main documentation](https://www.algolia.com/doc/) of Algolia.
+
+// todo: button
```
Comment (Haroenv) [inline _posts/2017-05-09-searching-for-packages.md]: make the page for signing up

File: _posts/2017-05-09-searching-for-packages.md [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+
+![importing a package with atom npm autocomplete](https://camo.githubusercontent.com/53350e9c6d303f60101e1644605fe62f529e45f2/687474703a2f2f672e7265636f726469742e636f2f643576695542385859372e676966)
+
+This was made by two colleagues at Algolia, [Gianluca](https://twitter.com/proudlygeek) and [Raymond](https://twitter.com/rayrutjes) after having the idea during a Happy Hour. It allows you to search for packages inline in your editor, and install it with Yarn or npm.
+
+### jsDelivr search
+➡️ [live](https://www.jsdelivr.com)
+➡️ [GitHub](https://github.com/jsdelivr/www.jsdelivr.com)
+
+// todo: gif
```
Comment (Haroenv) [inline _posts/2017-05-09-searching-for-packages.md]: make a gif for jsDelivr

File: _posts/2017-05-09-searching-for-packages.md [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+![searching on jsDelivr]()
+
+jsDelivr is free CDN for all packages on npm. They have been using Algolia since before this index existed, but switched to reuse this index when they switched to their [new architecture]().
```
Comment (Haroenv) [inline _posts/2017-05-09-searching-for-packages.md]: link to new architecture of jsDelivr post

**Comment by vvo** [comment]
Will put this in gdocs

**Comment by vvo** [comment]
Let's collaborate on gdocs now

**Pull request closed without merging**